packages/datagateway-dataview/src/views/card/isis/isisInvestigationsCardView.component.tsx

```diff
@@ -195,52 +218,52 @@ const ISISInvestigationsCardView = (
         filterComponent: textFilter,
       }}
       description={{
-        label: 'Description',
+        label: t('investigations.details.summary'),
         dataKey: 'SUMMARY',
         filterComponent: textFilter,
       }}
       information={[
         {
           icon: <Fingerprint />,
-          label: 'Visit ID',
+          label: t('investigations.rb_number'),
```

Comment:
The correct field name for Visit ID is `visit_id` (see old labels) - "RB number" is actually the name field, but again this shouldn't be a translation key name, instead it should be the value of the name key.